projects/bazel-rules-fuzzing-test-java/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN git clone -b java-oss-fuzz-2 https://github.com/fmeum/rules_fuzzing.git
```

Comment:
I believe this would need to be changed to the main repo, right?

Reply:
Yes, I will change it back to the main repo now that you have had a chance to try it out.

Reply:
Done.